src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -34,16 +34,16 @@ public class WalletController : Controller
         private readonly CoinType coinType;
 
         /// <summary>Specification of the network the node runs on - regtest/testnet/mainnet.</summary>
-        private readonly Network network;
+        public readonly Network Network;
```

Comment:
How come a lot of readonly properties have become public?

Reply:
It is because SmartContractWalletController now inherits from WalletController :)

Reply:
Then use protected in that case.

Reply:
Ok as discussed undo all WalletController stuff and only delete SC related code.